src/shared/sam/debugger/csharpSamDebug.ts

```diff
@@ -145,6 +169,27 @@ async function _installDebugger(
     }
 }
 
+async function downloadInstallScript(debuggerPath: string): Promise<string> {
+    let installScriptUrl: string
+    if (os.platform() == 'win32') {
+        installScriptUrl = 'https://aka.ms/getvsdbgps1'
+    } else {
+        installScriptUrl = 'https://aka.ms/getvsdbgsh'
```

Comment:
ternary is

1. easier to read
2. avoids `let` in favor of `const`
3. avoids unnecessary indentation/nesting.

Reply:
Since we now have 2 things to do per OS, i kept if-else